plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -684,7 +685,7 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
                auto missing = chain.missing_votes(id, qc_ext.qc);
                log_missing_votes(block, id, missing);
             }
-         } else if (block->is_proper_svnn_block()) {
+         } else if (block->is_proper_svnn_block() && (now - block->timestamp < fc::minutes(5) || (block->block_num() % 1000 == 0)) ) {
```

Comment:
What's purpose of the timestamp check here? Should `<` be `>=`?

Reply:
The check is so you don't spam the log while syncing.

Reply:
So why not log when the elapse time (now - block->timestamp) is greater than 5 minutes? With existing `now - block->timestamp < fc::minutes(5)`, you log every time when the difference is less than 5 minutes.

Reply:
now  - old block < 5 minutes
10:10:10.340 - 05:05:05.000 < 5 minutes
Only logs if the delta is small.